### PR TITLE
Move code to common module - Spring Data JPA

### PR DESCRIPTION
**Pull request by harry9656**

### CODE REVIEW

File: persistence-modules/spring-data-jpa-simple/src/main/java/com/baeldung/jpa/schemageneration/HibernateUtil.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package com.baeldung.jpa.schemageneration;
+
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
```
[the remark this answers is not in the document]
Comment (alvin577austria) [reply persistence-modules/spring-data-jpa-simple/src/main/java/com/baeldung/jpa/schemageneration/HibernateUtil.java:6]: I noticed there are a lot of unused imports across files, I think that is fine as long as the unused imports are not shown in the articles themselves. Can you please check? Thanks!

**Review by alvin577austria — CHANGES_REQUESTED** [review]
Some comments.

File: persistence-modules/spring-data-jpa-simple/src/main/java/com/baeldung/jpa/schemageneration/HibernateUtil.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (harry9656) [reply persistence-modules/spring-data-jpa-simple/src/main/java/com/baeldung/jpa/schemageneration/HibernateUtil.java:6]: I Removed unused imports from all the classes.
I double-checked the articles, but we don't show imports inside the articles anyway :)

**Review by alvin577austria — COMMENTED** [review]
Please move spring-boot-simple inside spring-boot-modules

**Pull request merged**